GoogleForCreators/web-stories-wp
# Media: Add initial Tenor stickers support

## Context



We've added support for the sticker content type for Tenor queries in our media3p API, which means we can now include it in the editor.

## Summary



Adds initial support for Tenor stickers to the 3p media library; behind a feature flag so that we can better test this.

## Relevant Technical Choices



## To-do



* More tests perhaps?
* Need to figure out if we wanna support both GIFs and WebP (e.g. using `<picture>`)
  Animated WebP images are usually larger than GIFs but also better quality, so it's a bit of a tradeoff. Need to pick one.
  Related: #11882

## User-facing changes



After enabling the Stickers experiment, there's a new "Stickers" tab in the 3p media library:

![Screenshot 2022-07-06 at 09 26 32](https://user-images.githubusercontent.com/841956/177535523-527af808-c366-4fa1-969a-c6176584148e.png)

https://user-images.githubusercontent.com/841956/177782397-89995b2b-fbf8-4c0b-81eb-f88f9cccca6d.mov


## Testing Instructions






This PR can be tested by following these steps:

1. Enable Stickers experiment
2. Test that functionality to list and search stickers works as expected
3. Stickers should be transparent and animated as expected, also on the preview


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #11878
